scripts/centos_jswarm.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
+
+CINCH="$(readlink -f "$(dirname "$0")/../")"
+CENTOS_VERSION="${1}"
+INVENTORY="${CINCH}/inventory/cent${CENTOS_VERSION}_jswarm_docker"
+if [ ! -e "${INVENTORY}" ]; then
+	echo "You must specify a CentOS version to continue"
+	exit 1
+fi
+# Extracts the version line from the setup.py script, and trims off the rest of the line to leave
+# only the expected version
```

Comment:
It literally can be anything. It's .* in the regex. I don't think, technically, there's any limitations on what the version string can be (we're following best practices and going with semver for now, but there's no need or requirement that we do, and this code supports literally anything).